cleanlab/filter.py

```diff
@@ -236,50 +235,42 @@ def multiclass_crossval_predict(pyx, labels):
     return pred
 
 
-def get_noise_indices(
-        s,
-        psx,
-        inverse_noise_matrix=None,
-        confident_joint=None,
-        frac_noise=1.0,
-        num_to_remove_per_class=None,
-        prune_method='prune_by_noise_rate',
-        sorted_index_method=None,
-        multi_label=False,
-        n_jobs=None,
-        verbose=0,
-):
-    """Returns the indices of most likely (confident) label errors in s. The
+def find_label_issues(labels, psx, confident_joint=None, frac_noise=1.0,
```

Comment:
I ran some (small-scale) experiments with deterministic underlying X->Y relationship, large sample-size, accurate model, low noise rate, and uniform (non-class-conditional) label noise (ie. the easiest possible setting).  

I found default value of `frac_noise = 1` tends to give much better recall than  precision in detecting label errors. However reducing `frac_noise` to say 0.5 or  0.9 did not boost the precision much, indicating there's hopefully big gains  to made by improving the label-quality-scoring functions. If `frac_noise>1` values were allowed, I wonder if recall could be further boosted without harming precision...

@JohnsonKuan

Reply:
This should be noted in project tracking and will likely not be addressed in this PR. @JohnsonKuan @jwmueller

Reply:
I created a new [Jira ticket ](https://cleanlab.atlassian.net/jira/software/projects/CLOS/boards/3?selectedIssue=CLOS-19) for this so we can revisit.